src/Components/FilterCard/FilterCard.js

```diff
@@ -32,13 +34,15 @@ export default function FilterCard(){
           <legend>Preferences</legend>
           <label>
             <input
+            className={styles.radioBtn}
               type="checkbox"
               onClick={() => dispatch({ type: "OUTOFSTOCK" })}
             />
             inclue out of stock products
           </label>
           <label>
             <input
+            className={styles.radioBtn}
               type="checkbox"
               onClick={() => dispatch({ type: "WITHFASTDELIVERY" })}
```

Comment:
it's better to use 'underscore' between two words. eg: "WITH_FAST_DELIVERY"

Reply:
done